File: FibonnaciSequence/Fibonnaci.java
import java.util.List;
import java.util.ArrayList;

public class Fibonnaci {

  private static List< Integer > fibonnaci( int value ) {
    final List< Integer > sequence = new ArrayList< Integer >();

    for ( int number = 0; number <= value ; number++ ) {
      if ( number <= 1 ) {
        sequence.add( number );
      } else {
        final int length = sequence.size();
        sequence.add( sequence.get( length - 1 ) + sequence.get( length - 2 ) );
      }
    }

    return sequence;
  }

  private static final int LIMIT = 44;

  public static void main( String[] arguments ) {
    final List< Integer > sequence = new ArrayList< Integer >();
    for ( int i = 0; i <= 25000; i++ ) {
      sequence.addAll( fibonnaci( LIMIT ) );
    }

    for ( int index = 0; index < sequence.size(); index++ ) {
      final int value = sequence.get( index );
      System.out.println( String.format( "%02d: %9d", index % ( LIMIT + 1 ), value ) );
    }
  }

}

File: FibonnaciSequence/Fibonnaci.cs
using System;
using System.Collections.Generic;

public class Fibonnaci
{

  private static List< int > fibonnaci( int value )
  {
    List< int > sequence = new List< int >();

    for ( int number = 0; number <= value ; number++ )
    {
      if ( number <= 1 )
      {
        sequence.Add( number );
      }
      else
      {
        int length = sequence.Count;
        sequence.Add( sequence[ length - 1 ] + sequence[ length - 2 ] );
      }
    }

    return sequence;
  }

  private const int LIMIT = 44;

  public static void Main( string[] arguments )
  {
    List< int > sequence = new List< int >();
    for ( int i = 0; i <= 25000; i++ )
    {
      sequence.AddRange( fibonnaci( LIMIT ) );
    }

    for ( int index = 0; index < sequence.Count; index++ )
    {
      int value = sequence[ index ];
      Console.WriteLine( string.Format( "{0:D2}: {1,9:D}", index % ( LIMIT + 1 ), value ) );
    }
  }

}
